tests/Fixtures/JunitFormat/features/bootstrap/AbortOnPHPErrorContext.php

```diff
@@ -0,0 +1,26 @@
+<?php
+use Behat\Behat\Context\Context;
+use Behat\Step\Given;
+use Behat\Step\Then;
+use Behat\Step\When;
+
+class Foo {}
+class AbortOnPHPErrorContext implements Context
+{
+    private $value;
+
+    #[Given('/I have entered (\d+)/')]
+    public function iHaveEntered($num) {
+        $this->value = $num;
+    }
```

Comment:
There's a lot of duplication in these step implementations between suites, which made sense when we were writing separate files as the only file for the scenario, but now I think adds to the noise and makes it harder to see what's actually triggering the behaviour we're testing. Maybe it would make sense to pull them all into a shared context?